samples/samples-react/public/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
```

Comment:
samples/samples-react/public/logo192.png
and samples/samples-react/public/logo512.png

are not need anymore, could I ask you to delete?